### PR TITLE
split signin into login and signup

### PR DESCRIPTION
**Pull request by giancarlopernudisegura**

### CODE REVIEW

File: server/routes/web.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+load_dotenv()
+
+HOST = os.getenv("FLASK_HOST")
+
+bp = Blueprint("webapp", __name__, url_prefix="/app")
+
+
+def frontend_page() -> Response:
+    current_app.send_static_file("index.html"), httpStatus.OK
```
Comment (dalidia) [inline server/routes/web.py]: You're missing a `return` statement

**Review by dalidia — CHANGES_REQUESTED** [review]
There's an issue when returning the index.html

File: server/routes/web.py (hunk 1 above)
Comment (giancarlopernudisegura) [reply server/routes/web.py]: Good catch! Don't know how it worked for me considering that `return` was missing.

**Review by dalidia — APPROVED** [review]
Looks good!

**Pull request merged**